firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/FirebaseAppDistributionTesterApiClient.java

```diff
@@ -97,12 +129,23 @@ AppDistributionReleaseInternal fetchNewRelease(
         connection.disconnect();
       }
     }
-
+    LogWrapper.getInstance().v(String.format("Response (%d): %s", responseCode, responseBody));
     if (!isResponseSuccess(responseCode)) {
       throw getExceptionForHttpResponse(responseCode);
     }
+    return responseBody;
+  }
 
-    return parseNewRelease(responseBody);
+  private String makeGetRequest(String path, String token) throws FirebaseAppDistributionException {
+    String url = String.format("https://%s/%s", APP_TESTERS_HOST, path);
+    HttpsURLConnection connection;
+    try {
+      connection = openHttpsUrlConnection(url, token);
+    } catch (IOException e) {
+      throw new FirebaseAppDistributionException(
+          ErrorMessages.NETWORK_ERROR, Status.NETWORK_FAILURE, e);
+    }
+    return readResponse(connection);
```

Comment:
this should probably be inside the try block. Then you don't need to catch the IOException inside readResponse() - just declare that method to throw IOException - it'll be handled here